samples/src/Microsoft.Azure.IIoT.App/src/Pages/Publishers.razor

```diff
@@ -52,8 +52,8 @@
                         @{ string MaxWorkers = CommonHelper.None;}
                         @if (publisher.Configuration != null) {
                             Capabilities = publisher.Configuration.Capabilities == null ? CommonHelper.None : publisher.Configuration.Capabilities.ToString();
-                            HeartbeatInterval = !IsTimeIntervalSet(publisher.Configuration.HeartbeatInterval) ? CommonHelper.None : publisher.Configuration.HeartbeatInterval.Value.TotalSeconds.ToString();
-                            JobCheckInterval = !IsTimeIntervalSet(publisher.Configuration.JobCheckInterval) ? CommonHelper.None : publisher.Configuration.JobCheckInterval.Value.TotalSeconds.ToString();
+                            HeartbeatInterval = !IsTimeIntervalSet(publisher.Configuration.HeartbeatInterval) ? CommonHelper.None : publisher.Configuration.HeartbeatInterval.Value.TotalSeconds.ToString() + " sec";
```

Comment:
Should we use the abbreviation "sec", or the unit "s"?

Reply:
To be consistent with other pages it's ok to use the abbreviation 'sec'